scripts/helpers/fetch-wp-api.js

```diff
@@ -0,0 +1,125 @@
+import { unescapeHTML } from './text-helpers';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Returns the fetch callback function for getting data from WP API.
+ *
+ * @param {string} endpoint Endpoint to fetch from (usually post type or taxonomy).
+ * @param {object} [options={}] Additional options for fine tunning.
+ * 
+ * @param {function} processId Function that allows custom id processing.
+ * @param {function} processLabel Function that allows custom select option label processing.
+ * @param {integer} [perPage=30] Define max perPage items to fetch.
+ * @param {string} [routePrefix='wp/v2'] Define if using custom or native WP routes.
+ * @param {object} [additionalParam={}] Define additional query params to fetch.
+ * @param {string} [cacheTime='86400000'] Define amount of time the cache is stored in seconds. Default 1 day
+ * @param {fields} [fields='id,title'] Fields to return for optimised response.
```

Comment:
Please align these so the descriptions match up and put a ` - ` before the description.

e.g.
```
@param {function} processId Function that allows custom id processing.
@param {function} processLabel Function that allows custom select option label processing.
 ```
 becomes
 ```
@param {function} processId    - Function that allows custom id processing.
@param {function} processLabel - Function that allows custom select option label processing.
 ```